pkg/scheduler/schedule_one.go

```diff
@@ -987,7 +987,7 @@ func (sched *Scheduler) handleSchedulingFailure(ctx context.Context, fwk framewo
 		logger.V(2).Info("Unable to schedule pod; no fit; waiting", "pod", klog.KObj(pod), "err", errMsg)
 	} else if apierrors.IsNotFound(err) {
 		logger.V(2).Info("Unable to schedule pod, possibly due to node not found; waiting", "pod", klog.KObj(pod), "err", errMsg)
-		if errStatus, ok := err.(apierrors.APIStatus); ok && errStatus.Status().Details.Kind == "node" {
+		if errStatus, ok := err.(apierrors.APIStatus); ok && (errStatus.Status().Details.Kind == "nodes" || errStatus.Status().Details.Kind == "node") { // Ususally `node` is used as the Kind, but `nodes` is also possible.
```

Comment:
Looks really weird, please explain when which one is used. I'd like to seek a possible solution where only one kind is here.

Reply:
At least, `apierrors` uses resource name(`nodes`) as a kind in `StatusError`.
https://github.com/kubernetes/kubernetes/blob/d61cbac69aae97db1839bd2e0e86d68f26b353a7/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go#L144-L156

I honestly don't know if there is room to plant StatusError from elsewhere.  `node` is nuanced enough that there may possibly be errors transmitted from user plug-ins, etc., which may not be needed.

Reply:
I don't like "keep it because someone may use it" in general. We want to catch non-found errors from kube-apiserver, so if kube-apiserver emits errors with "nodes", then supporting "nodes" only should be enough.

Reply:
👌 
d3def07550a